### ISSUE TITLE
Toolbar buttons not working on Android

### ISSUE THREAD

**Issue by udarts** [wix/react-native-zss-rich-text-editor#133]
Somehow the toolbar buttons are not working on android, they are working on iOS.
When I press for example, Bold, it is not selected, it does nothing.
Have tested in on multiple devices of android with the result.

I use the default setup as the example and like I said, it works on iOS.

Anyone having the same issue?

**Comment by FRizzonelli** [comment]
Yup totally the same, also the toolbar doesn't stay over the keyboard, so it's really a bad user experience

**Comment by sathishu37** [comment]
@udarts @FRizzonelli I have the same issue. any solution for this?

**Comment by FRizzonelli** [comment]
@sathishu37 I gave up on using this lib, seems that wix is really commited to fix/mantain other libraries (like RNN) and I'd say I totally agree to prioritize on other stuffs :)

**Comment by sathishu37** [comment]
@FRizzonelli  Thanks, Man.

**Comment by haven2world** [comment]
@sathishu37 Try this, it works for me.
`    "react-native-zss-rich-text-editor": "github:charpeni/react-native-zss-rich-text-editor.git#fix-selection-focus",`
